relay-kafka/src/producer/mod.rs

```diff
@@ -345,17 +263,10 @@ impl Producer {
         }
     }
 
-    fn sharded(shards: u64, producers: BTreeMap<u64, (String, Arc<ThreadedProducer>)>) -> Self {
-        Self {
-            last_report: Instant::now().into(),
-            inner: ProducerInner::Sharded(ShardedProducer { shards, producers }),
-        }
-    }
-
     /// Sends the payload to the correct producer for the current topic.
     fn send(
         &self,
-        organization_id: u64,
+        _organization_id: u64,
```

Comment:
This parameter is not needed anymore but it's piped through so much code that I was wondering if we should keep it for future use. If not, we can remove it.

Reply:
Let's remove it.